UKHomeOffice/sgmr-service
# SPCR-398 Fix width container so correct width is shown on desktop & mobile

### AC / Description of changes
- fixed html so it's no longer inserting a class in the wrong place
- removed old welcome banner files

### To Test
- view dashboard on a large screen
- view dashboard on a small screen
- > onboarding and welcome banners should no longer be indented on mobile

### Notes

---
* remember to merge to feature branches not master *
